wp-content/plugins/ig-content-loader-instance/ig-content-loader-instance-de_DE.po

```diff
@@ -0,0 +1,8 @@
+msgid "Page from other instance"
+msgstr "Seite aus Fremdinstanz"
+
+msgid "Select city"
+msgstr "Bitte Kommune ausw&auml;hlen"
+
+msgid "Please select"
```

Comment:
Fehlt hier nicht noch das "Please select (insert nothing)"? Oder kommt das in eine andere .po?